pages/EVENTS.md

```diff
@@ -0,0 +1,114 @@
+# Events
+
+If you’re interested in building web3 with us, catch up with Gno.land in person at one of our industry events. We’re looking to connect with developers and like-minded thinkers who can contribute to the growth of our platform.
+
+---
+
+## Upcoming Events
+
+<div class="columns-3">
+
+<div>
+
+### Eth Seoul
+
+- **The Evolution of Smart Contracts: A Journey into Gno.land**
+- Seoul, June 3, 2023
+- Manfred Touron
+
+[Learn more](https://2023.ethseoul.org/)
+
+</div>
+
+<div>
+
+### BUIDL Asia
```

Comment:
not upcoming anymore, same for eth seoul

Reply:
Yeah, I was waiting for Valeh to give me new updates for this page before committing the new one.